src/framework/execution/__tests__/dependencyGraph.test.ts

```diff
@@ -160,6 +160,64 @@ describe('executeStepDependencyGraph', () => {
     });
   });
 
+  test("waits for all of a steps's dependencies to complete prior to executing", async () => {
```

Comment:
`"` -> `'` or backtick?

Reply:
It's prettier's doing. https://prettier.io/docs/en/rationale.html#strings

Reply:
Oh, we don't have the `.prettierrc` file in the root of the project like we have in the `template/`